### PR TITLE
Show all jobs without algolia

### PR DESCRIPTION
**Pull request by cpjmcquillan**
Blank searches don't need a search, so don't do a search!

## Changes in this PR
- Fix a sort value bug
- Introducing PageObjects 🍾 
- Visit jobs page without algolia
- Welcome back to system specs for the jobs page (including pagination and sorting)

### CODE REVIEW

**Comment by martin-bangoura** [comment]
Review app deployed to <https://teaching-vacancies-review-pr-2796.london.cloudapps.digital>

File: app/services/search/search_builder.rb [hunk 1]
```diff
@@ -53,10 +56,15 @@ def build_suggestions
     end
   end
 
-  def call_algolia_search
-    search = Search::AlgoliaSearchRequest.new(search_params)
+  def call_search
+    search = if any?
```
Comment (david-mears-dfe) [inline app/services/search/search_builder.rb:60]: What does `any?` do? Google didn't enlighten me.

Comment (cpjmcquillan) [reply app/services/search/search_builder.rb:60]: It's a pre-existing method in the class if you expand above/below the diff

File: spec/page_objects/home.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+module PageObjects
```
Comment (alexbowen) [inline spec/page_objects/home.rb:1]: 🥳

**Review by alexbowen — APPROVED** [review]
good idea to do this

File: Gemfile [hunk 3]
```diff
@@ -45,6 +45,7 @@ gem "rollbar"
 gem "sanitize"
 gem "sidekiq"
 gem "sidekiq-cron"
+gem "site_prism"
```
Comment (cesidio) [inline Gemfile:48]: ❤️

**Review by cesidio — APPROVED** [review]

**Pull request merged**